modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,453 @@
+| `server_port` | `options.configMaps.tekton-results-api-config.data.config.SERVER_PORT`
+| `prometheus_port` | `options.configMaps.tekton-results-api-config.data.config.PROMETHEUS_PORT`
+| `gcs_bucket_name` | `options.configMaps.tekton-results-api-config.data.config.GCS_BUCKET_NAME`
+
+|===
++
+For the configuration parameters not listed in this table, use the regular specs as described in the documentation.
++
+[IMPORTANT]
+====
+Use the alternate parameter specs only if you need to use the additional settings in the `options.configMaps.tekton-results-api-config.data.config` section of the `TektonResult` CR.
+====
+
+* With this update, you can use the {tekton-results} API to retrieve the Go profiling data for {tekton-results}.
+
+* Before this update, {tekton-results} checked the user authentication when displaying every fragment of log data. With this update, {tekton-results} checks the user authentication only once per log data request. This change improves the response time for the {tekton-results} log API, which is used for displaying logs using the command line utility.
+
+[id="breaking-changes-1-15_{context}"]
+== Breaking changes
+
+* Before this update, {pac} set the `git-provider`, `sender`, and `branch` labels in a pipeline run. With this update, {pac} no longer sets these labels, Instead, it sets the `pipelinesascode.tekton.dev/git-provider`, `pipelinesascode.tekton.dev/sender`, and `pipelinesascode.tekton.dev/branch` annotations.
+
+* With this update, you can no longer use the `jaeger` exporter for OpenTelemetry tracing. You can use the `oltptraceexporter` for tracing.
+
+[id="known-issues-1-15_{context}"]
+== Known issues
+
+* The new `skopeo-copy` task, which is available from the `openshift-pipelines` namespace by using the cluster resolver, does not work when the `VERBOSE` parameter is set to `false`, which is the default setting. As a workaround, when you use this task, set the `VERBOSE` parameter to `true`. The issue does not apply to the `skopeo-copy` `ClusterTask`.
+
+* The new `skopeo-copy` task, which is available from the `openshift-pipelines` namespace by using the cluster resolver, fails when you use it to push or pull an image to or from an {OCP} internal image repository, such as `image-registry.openshift-image-registry.svc:5000`. As a workaround, set the `DEST_TLS_VERIFY` or `SRC_TLS_VERIFY` parameter to `false`. Alternatively, use an external image repository that has a valid SSL certificate. The issue does not apply to the `skopeo-copy` `ClusterTask`.
+
+* The new `s2i-*` tasks, which are available from the `openshift-pipelines` namespace by using the cluster resolver, fail if you clone a Git tepository to a subdirectory of the `source` workspace and then set the `CONTEXT` parameter of the task.As a workaround, when you use these tasks, do not set the `CONTEXT` parameter. The issue does not apply to the `s2i-*` `ClusterTasks`.
```

Comment:
@ppitonak do you see this as workaround i mean not setting the CONTEXT param or something else will be more meaningful